### DISCUSSION TITLE
Do you use any formatter for javascript?

### DISCUSSION THREAD

**Discussion by eyllanesc** [HEIGE-PCloud/DoIt#369] [Q&A] [answered]
Maybe you should point out a developer guide such as tools. When I implemented my PRs I had a hard time keeping the javascript format that you use.

**Comment by HEIGE-PCloud** [comment, marked as answer]
An ESLint config has been introduced in #368

![image](https://user-images.githubusercontent.com/52968553/144570211-1f3d95f6-a26b-42e2-b293-3153a7c719e1.png)

But it will definitely take some time to format the code...
